tests/integration/test_flight_interface_worker.py

```diff
@@ -0,0 +1,70 @@
+"""
+Flight interface worker integration test
+"""
+
+import multiprocessing as mp
```

Comment:
you don't need to create a multiprocessing worker here, since were only running once process it can be done in the main process